bitnami/external-dns/templates/dep-ds.yaml

```
@@ -31,6 +31,7 @@ spec:
   template:
     metadata:
       labels: {{ include "external-dns.labels" . | nindent 8 }}
+      {{- if or .Values.podAnnotations (and .Values.metrics.enabled .Values.metrics.podAnnotations) (include "external-dns.createSecret" .) (and (eq .Values.provider "designate") .Values.designate.customCA.enabled) }}
       annotations:
         {{- if or .Values.podAnnotations .Values.metrics.enabled }}
```

Thank you for your contribution! Could you please update this line to match the conditional you added?
```suggestion
        {{- if or .Values.podAnnotations (and .Values.metrics.enabled .Values.metrics.podAnnotations) }}
```

---

I did not add it because effectively the result will be the same due to the condition already being in the template helper, but it also doesn't hurt putting it here.